test/models/services.ts

```diff
@@ -831,5 +831,49 @@ describe('createWorkflowSteps', function () {
     it('creates a third and final workflow step for the shapefile subsetter', function () {
       expect(steps[2].serviceID).to.equal('shapefile subsetter');
     });
+
+    it('first step uses artifact bucket as the staging location', function () {
```

Comment:
Nit: the `it` is supposed to be read as if part of a sentence - so something like
```
it('uses the artifact bucket as the staging location for the first step',
```